### PR TITLE
feat(nag): refactored nag site

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
Refactored the `nag` module site to make use of the new features as of 
https://github.com/fomantic/Fomantic-UI/pull/1795 and
https://github.com/fomantic/Fomantic-UI/pull/1823

## Screenshot
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by exoego — APPROVED** [review]
Document LGTM 👍 

I think Nag's default foreground and background color seems low-contrast, so do not satisfy some accessibility guideline.
https://color.a11y.com/ContrastPair/?bgcolor=7E7E7E&fgcolor=000000
https://color.a11y.com/ContrastPair/?bgcolor=7E7E7E&fgcolor=ffffff

Maybe other module also have same contrast issues.

**Comment by lubber-de** [comment]
@exoego I also had some concerns about the contrast. Thats why i already slightly increased the background grey to make it more readable but keep the original SUI look in the first PR. (the original value was 666666 which is even worse)

The original SUI Docs were only showing the variants with a title, which is white, thus readable. 
If we increase the default grey too much, then the white title (which is probably always used in existing projects, because all original SUI examples were done this way) is going to be unreadable.

Setting the default text color from black to also white instead (so it looks just like the title) would perhaps visually break existing code...however i pretty much doubt people used it without the title variant or even used their own CSS Styling anyway.

I think the maximum change for the backgroundcolor is #888888 or #909090 to also keep the white title readable 

### Background Comparison (it's a slow gif... be patient)
nagbackground

Should i create a PR for that? Or should we cover this separately after investigating into all compoents as suggested by you in the other issue?

File: server/documents/modules/nag.html.eco [hunk 1]
```diff
@@ -7,72 +7,502 @@ standalone  : true
...
+      </div>
+
 
   </div>
 
+  <div class="ui tab" data-tab="usage">
+      <h2 class="ui dividing header">Usage</h2>
+
+      <div class="no example">
+        <h4 class="ui header"></h4>
```
Comment (ko2in) [inline server/documents/modules/nag.html.eco:220]: This empty header creates a lot of gap between the above header "Usage" and the example.

Comment (lubber-de) [reply server/documents/modules/nag.html.eco]: Fixed by 9d393cdf3a133a6c6f9f31317ce64454de6e4955
Please review again

**Review by ko2in — APPROVED** [review]
LGTM.

**Comment by lubber-de** [comment]
i had to fix a camelcase typo in the settings for `storageMethod` by [`b05fbfb` (#257)](https://github.com/fomantic/Fomantic-UI-Docs/pull/257/commits/b05fbfb8e075cbdb9235daef7c96e20d7cfc1ba6)

Please review again

**Review by ko2in — APPROVED** [review]
LGTM.

**Review by exoego — APPROVED** [review]
LGTM

**Comment by lubber-de** [comment]
@exoego Please also review/approve https://github.com/fomantic/Fomantic-UI/pull/1823 the implemented callbacks are described in this Docs PR :wink:
Thanks

**Pull request merged**